### PR TITLE
Blog Post: Third Wave of Nostr Grants

### PR DESCRIPTION
**Pull request by dergigi**
Ready for review. I'd love to publish it today.

---

Preview: [/blog/nostr-grants-october-2023](https://os-website-git-3rd-wave-nostr-grants-open-sats.vercel.app/blog/nostr-grants-october-2023)

Social Preview:

Screenshot 2023-10-18 at 12 04 18

### CODE REVIEW

File: data/blog/nostr-grants-october-2023.mdx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+title: "Third Wave of Nostr Grants"
+date: '2023-10-18'
+tags: ['OpenSats', 'grants', 'nostr']
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/11-nostr-oct.jpg']
+summary: "OpenSats is funding another dozen open-source projects in the nostr ecosystem."
+---
+
+You know what the best thing about nostr is? It's not its simple architecture.
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:11]:
```suggestion
You know what the best thing about Nostr is? It's not its simple architecture.
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+License: AGPL-3.0
+
+## Nostur
+
+[Nostur](https://nostur.com/) is a feature-rich nostr client for iOS that aims
+to be simple to use for Mac, iPhone, and iPad users. It supports multiple
+accounts, has nsecBunker support, and imposter detection features. Nostur is
+opinionated in its
+[mission](https://blogstack.io/naddr1qq8xummnw36hyttdd9ehx6t0dcq3gamnwvaz7tmjv4kxz7fwv3sk6atn9e5k7q3qn0sturny6w9zn2wwexju3m6asu7zh7jnv2jt2kx6tlmfhs7thq0qxpqqqp65w8jxczy)
+and uses Apple-native features such as iCloud to onboard as many users to nostr
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:94]:
```suggestion
and uses Apple-native features such as iCloud to onboard as many users to Nostr
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+- Blowater by [Water
+Blower](https://njump.me/npub1dww6jgxykmkt7tqjqx985tg58dxlm7v83sa743578xa4j7zpe3hql6pdnf)
+- Nostur by
+[Fabian](https://njump.me/npub1n0sturny6w9zn2wwexju3m6asu7zh7jnv2jt2kx6tlmfhs7thq0qnflahe)
+- Nostrocket by
+[@gsovereignty](https://njump.me/npub1mygerccwqpzyh9pvp6pv44rskv40zutkfs38t0hqhkvnwlhagp6s3psn5p)
+- snort.social by
+[Kieran](http://npub1v0lxxxxutpvrelsksy8cdhgfux9l6a42hsj2qzquu2zk7vc9qnkszrqj49)
+- zap.stream by
+[Kieran](http://npub1v0lxxxxutpvrelsksy8cdhgfux9l6a42hsj2qzquu2zk7vc9qnkszrqj49)
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:46]:
```suggestion
[Kieran](https://njump.me/npub1v0lxxxxutpvrelsksy8cdhgfux9l6a42hsj2qzquu2zk7vc9qnkszrqj49)
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+- Nozzle by
+[@dluvian](https://njump.me/npub1useke4f9maul5nf67dj0m9sq6jcsmnjzzk4ycvldwl4qss35fvgqjdk5ks)
+- Blowater by [Water
+Blower](https://njump.me/npub1dww6jgxykmkt7tqjqx985tg58dxlm7v83sa743578xa4j7zpe3hql6pdnf)
+- Nostur by
+[Fabian](https://njump.me/npub1n0sturny6w9zn2wwexju3m6asu7zh7jnv2jt2kx6tlmfhs7thq0qnflahe)
+- Nostrocket by
+[@gsovereignty](https://njump.me/npub1mygerccwqpzyh9pvp6pv44rskv40zutkfs38t0hqhkvnwlhagp6s3psn5p)
+- snort.social by
+[Kieran](http://npub1v0lxxxxutpvrelsksy8cdhgfux9l6a42hsj2qzquu2zk7vc9qnkszrqj49)
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:44]:
```suggestion
[Kieran](https://njump.me/npub1v0lxxxxutpvrelsksy8cdhgfux9l6a42hsj2qzquu2zk7vc9qnkszrqj49)
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+License: GPL-3.0
+
+## Nostrocket
+
+Nostrocket is a new economic tool that uses Bitcoin and Nostr to create
+leaderless, location-independent economies that reward productive solutions to
+shared problems. Technically speaking, a Nostrocket client is a state machine.
+Different parts of the state machine can run in different places, e.g., much of
+the state can be rebuilt directly in the browser, but computationally heavy
+operations like indexing a large number of nostr events or validating the
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:109]:
```suggestion
operations like indexing a large number of Nostr events or validating the
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+Bitcoin blockchain (some parts of the protocol also require knowledge of the
+current state of Bitcoin) are better done closer to the metal.
+
+Repository: https://github.com/nostrocket
+
+License: MPL-2.0
+
+## snort.social
+
+Snort is a web client with the goal of making nostr's features accessible in a
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:119]:
```suggestion
Snort is a web client with the goal of making Nostr's features accessible in a
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+user-friendly manner while maintaining its censorship-resistant properties.
+Snort's primary focus is to support a wide variety of features that showcase the
+power of the nostr network while remaining a clean and performant client.
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:122]:
```suggestion
power of the Nostr network while remaining a clean and performant client.
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+
+Repository: https://git.v0l.io/Kieran/snort
+
+License: MIT
+
+## zap.stream
+
+[Zap.stream](https://zap.stream/) is a live streaming nostr client PWA that
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:130]:
```suggestion
[Zap.stream](https://zap.stream/) is a live streaming Nostr client PWA that
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+allows people to interact with externally hosted HLS streams. Users can chat,
+zap, and react to the stream while watching a live stream. The goal of
+[zap.stream](https://zap.stream/) is to create a nostr alternative to
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:133]:
```suggestion
[zap.stream](https://zap.stream/) is to create a Nostr alternative to
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+[twitch.tv](https://www.twitch.tv/) and similar online streaming platforms.
+
+Repository: https://git.v0l.io/Kieran/stream
+
+License: MIT
+
+## algia
+
+Algia is a CLI client that can manipulate nostr operations such as post, like,
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:142]:
```suggestion
Algia is a CLI client that can manipulate Nostr operations such as post, like,
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 11]
```diff
@@ -0,0 +1,223 @@
...
+boost, delete, as well as streams. Algia can handle JSON, so you can easily
+create nostr bots with external tools such as `jq`.
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:144]:
```suggestion
create Nostr bots with external tools such as `jq`.
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 12]
```diff
@@ -0,0 +1,223 @@
...
+
+Repository: https://github.com/mattn/algia
+
+License: MIT
+
+## nostr.watch
+
+[nostr.watch](https://nostr.watch/) aims to be an authoritative source for nostr
+relay data. It aggregates data on nostr relays (and the network in general) to
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:153]:
```suggestion
relay data. It aggregates data on Nostr relays (and the network in general) to
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 13]
```diff
@@ -0,0 +1,223 @@
...
+architecture.
+
+Repository: https://github.com/dskvr/nostr-watch
+
+License: MIT
+
+## relay.tools
+
+[Relay.tools](https://relay.tools/) is a web-based control panel and deployment
+system for nostr relays. Its goal is to enable the entire nostr population to
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:165]:
```suggestion
system for Nostr relays. Its goal is to enable the entire Nostr population to
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 14]
```diff
@@ -0,0 +1,223 @@
...
+filtering rules, and best practice settings. It does not rely on any cloud
+provider and is 100% free from vendor lock-in.
+
+Repository: https://github.com/relaytools
+
+License: GPL-3.0
+
+## Nostree
+
+[Nostree](https://nostree.me/) is a nostr micro-app to create, manage, share,
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:177]:
```suggestion
[Nostree](https://nostree.me/) is a Nostr micro-app to create, manage, share,
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 15]
```diff
@@ -0,0 +1,223 @@
...
+and explore links and reference lists (as well as show notes and other stuff).
+The idea is that it becomes a good presentation card for you and a place to
+discover nostr & the wider internet in a more accessible way.
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:180]:
```suggestion
discover Nostr & the wider internet in a more accessible way.
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 16]
```diff
@@ -0,0 +1,223 @@
...
+
+Repository: https://github.com/gzuuus/linktr-nostr
+
+License: GPL-3.0
+
+## ONOSENDAI - Cyberspace Meta-Protocol
+
+[ONOSENDAI](https://www.onosendai.tech/) is an experimental nostr client that
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:188]:
```suggestion
[ONOSENDAI](https://www.onosendai.tech/) is an experimental Nostr client that
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 17]
```diff
@@ -0,0 +1,223 @@
...
+maps and visualizes the nostr protocol in a 3D space. The goal of ONOSENDAI is
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:189]:
```suggestion
maps and visualizes the Nostr protocol in a 3D space. The goal of ONOSENDAI is
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 18]
```diff
@@ -0,0 +1,223 @@
...
+to be the reference implementation of a novel [cyberspace
+protocol](https://github.com/arkin0x/cyberspace/), a digital reality that,
+thanks to Bitcoin and nostr, stands on its own. You can play around with an
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:192]:
```suggestion
thanks to Bitcoin and Nostr, stands on its own. You can play around with an
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 19]
```diff
@@ -0,0 +1,223 @@
...
+early version of the client at [onosendai.tech](https://www.onosendai.tech/).
+
+Repository: https://github.com/arkin0x/cyberspace/
+
+License: CC-BY-SA / CC-BY-NC-ND
+
+## 0xchat
+
+0xchat is a secure chat client for iOS and Android built on top of the nostr
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:201]:
```suggestion
0xchat is a secure chat client for iOS and Android built on top of the Nostr
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 20]
```diff
@@ -0,0 +1,223 @@
...
+
+Repository: https://github.com/0xchat-app
+
+License: MIT / LGPL-3.0
+
+---
+
+Nostr is not fucking leaving, and neither are we.
+
+Support for the nostr projects above is made possible by generous donors like
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:218]:
```suggestion
Support for the Nostr projects above is made possible by generous donors like
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 21]
```diff
@@ -0,0 +1,223 @@
+---
+title: "Third Wave of Nostr Grants"
+date: '2023-10-18'
+tags: ['OpenSats', 'grants', 'nostr']
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/11-nostr-oct.jpg']
+summary: "OpenSats is funding another dozen open-source projects in the nostr ecosystem."
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:8]:
```suggestion
summary: "OpenSats is funding another dozen open-source projects in the Nostr ecosystem."
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 22]
```diff
@@ -0,0 +1,223 @@
...
+You know what the best thing about nostr is? It's not its simple architecture.
+It's not that it has cryptographic proof baked into every action. It's not that
+every note stands on its own. It's not the good vibes, the zaps, or all the
+"other stuff," or the fact that it's officially called [THE
+NOSTR](https://njump.me/note1gt4q3wdd6lfjzmweh9ge594s8vjga5cshuqdqzdhj9d6ahdlgasqpe96fa)
+(yes, in all caps). It's that it works. And that it has worked since inception
+and that it will (~~probably~~ ~~maybe~~ very likely) continue to work in the future.
+
+And while it will continue to work, it will improve continually, as open
+protocols tend to do. It becomes more obvious by the day that nostr is here to
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:20]:
```suggestion
protocols tend to do. It becomes more obvious by the day that Nostr is here to
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 23]
```diff
@@ -0,0 +1,223 @@
...
+stay, and we're going to do our best to make it the best, weirdest protocol it
+can be.
+
+We are happy to announce 12 new grants for open-source projects in the nostr
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:24]:
```suggestion
We are happy to announce 12 new grants for open-source projects in the Nostr
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 24]
```diff
@@ -0,0 +1,223 @@
...
+ecosystem. Adding to the [16 grants announced in
+July](https://opensats.org/blog/nostr-grants-july-2023), plus the [three grants
+announced in
+August](https://opensats.org/blog/bitcoin-and-nostr-grants-august-2023), the
+total number of nostr projects funded by OpenSats as of October is 31. [October
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:29]:
```suggestion
total number of Nostr projects funded by OpenSats as of October is 31. [October
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 25]
```diff
@@ -0,0 +1,223 @@
...
+It aims to provide the best DM experience on Nostr so far. Its long-term goal is
+to replace the likes of Telegram, Slack, Discord, and similar.
+
+Repository: https://github.com/BlowaterNostr/blowater
+
+License: AGPL-3.0
+
+## Nostur
+
+[Nostur](https://nostur.com/) is a feature-rich nostr client for iOS that aims
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:89]:
```suggestion
[Nostur](https://nostur.com/) is a feature-rich Nostr client for iOS that aims
```

File: data/blog/nostr-grants-october-2023.mdx [hunk 26]
```diff
@@ -0,0 +1,223 @@
...
+## 0xchat
+
+0xchat is a secure chat client for iOS and Android built on top of the nostr
+protocol. It prioritizes privacy and offers encrypted private chats & contacts,
+as well as multiple DM types: normal, sealed, and secret. The client offers
+audio and video calling capabilities between contacts, using Nostr relays as a
+signaling mechanism. The actual audio and video calls are facilitated via ICE
+servers, and users are free to select their own ICE servers if preferred. In the
+future, the client will support multi-person voice and video chats, expanding
+the communication possibilities of nostriches worldwide.
```
Comment (Arvin21M) [inline data/blog/nostr-grants-october-2023.mdx:208]:
```suggestion
the communication possibilities of Nostriches worldwide.
```

**Review by Arvin21M — CHANGES_REQUESTED** [review]
These projects are bad ass.

**Comment by dergigi** [comment]
Thanks for spotting the broken `npub` links @Arvin21M - appreciate it! 🙏 Fixed now.

**Comment by dergigi** [comment]
I think we'll keep the lowercase "nostr" though. Thanks for giving everything a read 🧡

**Comment by dergigi** [comment]
ACK by @modl21, will merge & publish this now.

**Pull request merged**